serverless.yml

```diff
@@ -26,6 +26,10 @@ provider:
         - "logs:CreateLogStream"
         - "logs:PutLogEvents"
       Resource: "arn:aws:logs:*:*:*"
+    environment:
```

Comment:
Got too many spaces here, this should be a level back indenting wise (along with the line below too)